lib/index.js

```diff
@@ -60,7 +59,7 @@ const td = new TextDecoder('utf8', {fatal: true, ignoreBOM: true})
  * sink.on('finish', () => console.log(n.toString('base64')))
  * process.stdin.pipe(sink)
  */
-class NoFilter extends stream.Transform {
+class NoFilter extends Transform {
```

Comment:
I think just putting an `export` here will fix the docs.

Reply:
I put an export at the bottom.

Have learned from lots of conversion from cjs to esm that it's just easier to review PRs if it is just a mater of changing
```diff
- module.exports = {
+ export {
   foo,
   bar
 }
```
think it's easier to see what gets exported instead of having multiple `export const [xyz]` everywhere